content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to get up and running with Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+To get started with Optimize, you will need the following:
+
+- A [Prisma Data Platform workspace](https://console.prisma.io).
+- A project that uses [Prisma Client](/orm/prisma-client) `5.16.1` or higher (we always recommend using the latest version of Prisma).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
+
+## 1. Enable Optimize
+
+Login to your [Prisma Data Platform account](https://console.prisma.io) and then navigate to the [Optimize dashboard](https://optimize.prisma.io).
+
+## 2. Add Optimize to your application
+
+### 2.1. Update your `schema.prisma` file
+
+In your Prisma schema's `generator` block add the `tracing` preview feature:
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["tracing"]
+}
+```
+
+Then generate Prisma Client using:
+```terminal
+npx prisma generate
+```
+
+### 2.2. Install the Optimize Prisma Client extension
+
+Install the latest version of Prisma Client and Optimize Prisma Client extension:
```

Comment:
Install the latest version of Prisma Client and the Optimize Prisma Client extension:
```​⬤